assets/js/scripts.js

```diff
@@ -104,8 +104,11 @@ $(document).ready(function () {
     $("#item3").empty();
     $("#item4").empty();
     $("#item5").empty();
+    $("#drinkHeading").empty();
     let drinkArray = ["one", "two", "three", "four", "five"];
```

Comment:
Is drinkArray used?

Reply:
no it's not. I just removed and pushed. Sorry I noticed that earlier and forgot to delete it then.